<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDialog" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223, 223, 167);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="bottomMargin">
    <number>130</number>
   </property>
   <item>
    <widget class="QLabel" name="label_admin">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>300</number>
     </property>
     <property name="rightMargin">
      <number>300</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_adusername">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="leftMargin">
      <number>300</number>
     </property>
     <property name="rightMargin">
      <number>300</number>
     </property>
     <property name="bottomMargin">
      <number>13</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_adpassword">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>550</number>
     </property>
     <property name="rightMargin">
      <number>550</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_adlogin">
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Log-In:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>admin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_adlogin</sender>
   <signal>clicked()</signal>
   <receiver>admin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
